### DISCUSSION TITLE
Vorta not visible when cmd-tabbing on macOS

### DISCUSSION THREAD

**Discussion by flukuschumska** [borgbase/vorta#1642] [FAQ] [answered]
I am using Vorta 0.8.10 on macOS 12.6.3 and it's running fine. But what really confuses me is that Vorta.app does not appear in the row of icons if I want to switch back to it after working in another application. Since I tend to work with maximised windows, I always have to go to the menu bar or to pull it back to the foreground with Spotlight.
Vorta is the only application I remember behaving like this. Why is it so, and can I change this behaviour?

**Comment by m3nu** [comment, marked as answer]
Intentional. You can change it here: https://github.com/borgbase/vorta/blob/master/package/vorta.spec#L74

**Reply by flukuschumska** [reply]
I am impressed, this was fast…
I simply changed /Applications/Vorta.app/Contents/Info.plist, and everything is fine.
Thank you very much and have a nice Sunday!
